docs/use-cases/defi/integrate-with-xrpl-dex.md

```diff
@@ -0,0 +1,105 @@
+
+
+## Managing Off-Ledger Assets (Optional)
+
+If you are setting up an exchange to hold custody of user funds, you must manage your own balance sheet to track customers' holdings. In this situation, the exchange's accounts are represented _on-chain_ and can be queried. All users of that exchange are handled _off-chain_ by the exchange's accounting system.
+
+XRP are represented on the XRP Ledger as an unsigned integer count of _drops_, where one XRP is 1,000,000 drops. Ripple recommends that your accounting software handles XRP balances in _drops_, but displays to users their balance in units of XRP.
+
+One drop (.000001 XRP) can't be further subdivided. Keep this in mind when calculating and displaying rates between XRP and other assets.
+
+To learn more about XRP and token specifics, see: [Currency Formats](currency-formats.html).
+
+
+## Trading in the DEX
+
+A trade in the XRP Ledger DEX is called an [Offer](offers.html). An Offer is effectively a [_limit order_](https://en.wikipedia.org/wiki/Order_(exchange)#Limit_order) to buy or sell a specific amount of one currency (XRP or a token) for a specific amount of another.
+
+To see how an example trade is made on the DEX, using the **JavaScript** or **Python** libraries, see: [Trade in the Decentralized Exchange](trade-in-the-decentralized-exchange.html).
+
+
+### Create Order Books
+
+Use the `book_offers` method to retrieve a list of offers (an order book) between two currencies on the XRP Ledger. The response omits unfunded `Offers` and reports how much of each remaining Offer's total is currently funded. This step isn't technically a requirement, but it's good practice to confirm the exchange rates of currencies, using the `quality` field.
+
+When searching for tokens, ensure you have the right `issuer` selected.
+
+See: [`book_offers`](book_offers.html).
+
+
+### Submitting Offers
+
+To creat ean offer, send an `OfferCreate` transaction.
+
+An `OfferCreate` transaction is an instruction to conduct a trade, either between two tokens, or a token and XRP. Each transaction contains a buy amount (`TakerPays`) and a sell amount (`TakerGets`). When the transaction is processed, it automatically consumes matching or crossing Offers to the extent possible. If that doesn't completely fill the new Offer, then the rest becomes an Offer object in the ledger.
+
+The Offer object waits in the ledger until other Offers or cross-currency payments fully consume it. The account that placed the Offer is called the Offer's _owner_. You can cancel your own Offers at any time, using the dedicated `OfferCancel` transaction, or as an option of the `OfferCreate` transaction.
+
+**Note:** Any Offer in the DEX that would exchange two tokens can use XRP as an intermediary currency in a synthetic order book. The XRPL DEX automatically uses this bridging technique to improve exchange rates and liquidity when doing so is cheaper than trading token-to-token. To learn more, see: [auto-bridging](autobridging.html).
```

Comment:
The callout syntax here should be updated.

```suggestion
{% admonition type="info" name="Note" %}
Any Offer in the DEX that would exchange two tokens can use XRP as an intermediary currency in a synthetic order book. The XRPL DEX automatically uses this bridging technique to improve exchange rates and liquidity when doing so is cheaper than trading token-to-token. To learn more, see: [auto-bridging](autobridging.html).
{% /admonition %}
```

(Note, my suggestion uses the old link, but that should also be updated.)